.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,26 @@
+name: Upload Python Package
```

Comment:
Nice addition. Makes lifer easier.

Reply:
@mjwen I think we already have it. @ellio167 is the admin and can add your credentials as well

Reply:
@mjwen Can you give me write access to kimpy? It makes it easier to assign and add labels or milestones, or ...

Reply:
I guess the admin of the repo @ellio167 can give you write access?